packages/editor/src/plugins/fill-in-the-blanks-exercise/components/blank-draggable-dummy-area.tsx

```diff
@@ -0,0 +1,62 @@
+import { faPlus } from '@fortawesome/free-solid-svg-icons'
+import { FaIcon } from '@serlo/frontend/src/components/fa-icon'
+import { useRef } from 'react'
+
+import { BlankDraggableDummyAnswer } from './blank-draggable-dummy-answer'
+import type { FillInTheBlanksExerciseProps } from '..'
+import { useEditorStrings } from '@/contexts/logged-in-data-context'
+
+interface BlankDraggableDummyAreaProps {
+  extraDraggableAnswers: FillInTheBlanksExerciseProps['state']['extraDraggableAnswers']
+}
+
+export function BlankDraggableDummyArea(props: BlankDraggableDummyAreaProps) {
+  const { extraDraggableAnswers } = props
+
+  const areaRef = useRef<HTMLDivElement>(null)
+  const blanksExerciseStrings = useEditorStrings().plugins.blanksExercise
+
+  const dummyValues = extraDraggableAnswers.map(({ answer }) => answer.value)
+
+  return (
+    <div className="mt-8 px-4" ref={areaRef}>
+      {dummyValues.length > 0 ? (
+        <>
+          {blanksExerciseStrings.dummyAnswers}:
+          <div className="flex flex-wrap">
+            {dummyValues.map((answer, index) => (
+              <BlankDraggableDummyAnswer
+                key={index}
+                text={answer}
+                onChange={(event) => {
+                  extraDraggableAnswers[index].answer.set(event.target.value)
+                }}
+                onKeyDown={(e) => {
+                  if (e.key === 'Enter') {
+                    extraDraggableAnswers.insert()
+                  }
+                }}
+                onRemoveClick={() => {
+                  extraDraggableAnswers.remove(index)
+                  // focus new last input to make sure we don't loose focus
+                  const allInputs = areaRef.current?.querySelectorAll('input')
+                  if (allInputs) {
+                    void [...allInputs]?.at(-2)?.focus()
```

Comment:
How does this work?

Reply:
Tbh, it's a part @elbotho added, maybe he can chip in 😁

Reply:
It's the nicest way I know to get the second to last entry in a list. I'm not getting the last one, because that's the one about to be deleted.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/at

Reply:
I considered writing a complex css query, but decided this was easier to understand. Maybe I was wrong :)

Reply:
Makes sense now! Thanks! :)